### DISCUSSION TITLE
New file definitions!

### DISCUSSION THREAD

**Discussion by raomin** [raomin/ESPAltherma#375] [Announcements]
Hi everyone!

First of all, happy new year to everyone, and happy birthday to ESPAltherma which turned 3 already!
I'm amazed to see the community that we have built together in only 3 years...

So, great news, I've updated the definition files. We have now new models supported and hopefully clearer filenames :)

Happy ESPAltherma to everyone!

Raomin

**Comment by G0b3** [comment]
Is it possible that the update made my AlthermaSensors Unavailable?
This night they became unavailable. 
I'm not sure how to fix this...

**Reply by G0b3** [reply]
It is fixed again..

**Comment by neildsb** [comment]
I found a problem that manually fixed - https://github.com/raomin/ESPAltherma/issues/379

**Comment by TRIROG** [comment]
@raomin  thank you for all the effort you are putting into this project and the good work!

Thanks for definition filename updates!
